src/core/Item.java

```diff
@@ -0,0 +1,115 @@
+package core;
+
+import commands.Command;
+
+import java.io.*;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Marcelo Loayza on 17/05/2016.
+ * Modified by Marcela Barrionuevo  added functionality for location, type, sizeBytes, sizeKB, sizeMB, sizeTB, filePermissions and filePermissions on 18/05/2016
+ */
+public class Item {
+
+    private String name;
+    private String location;
+    private String type;
+    private double sizeBytes;
+    private double sizeKB;
+    private double sizeMB;
+    private double sizeGB;
+    private double sizeTB;
+    private Date lastModified;
+    private String filePermissions;
+
+    private Map<String, String> map;
+
+    public Item (java.io.File file) {
```

Comment:
Why are you adding the whole package name for java.io.File if you have imported that package?